src/jazayeri_lab_to_nwb/watters/timeseries_interfaces.py

```diff
@@ -0,0 +1,207 @@
+        processing_module = get_module(
+            nwbfile=nwbfile, name='behavior', description=module_description)
+
+        # Add data to module
+        processing_module.add_data_interface(eye_position)
+
+        return nwbfile
+
+
+class PupilSizeInterface(TemporalAlignmentMixin):
+    """Pupil size interface."""
+
+    def __init__(self, folder_path: FolderPathType):
+        # Find pupil size file
+        folder_path = Path(folder_path)
+        pupil_size_file = folder_path / 'pupil_size_r.json'
+        assert pupil_size_file.exists(), f'Could not find {pupil_size_file}'
+        
+        # Load pupil size data and set data attributes
+        pupil_size_data = json.load(open(pupil_size_file, 'r'))
+        self.set_original_timestamps(np.array(pupil_size_data['times']))
+        self._pupil_size = np.array(pupil_size_data['values'])
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+        # Make SpatialSeries
+        pupil_size = TimeSeries(
+            name='pupil_size',
+            data=H5DataIO(self._pupil_size, compression='gzip'),
+            unit='pixels',
+            conversion=1.0,
+            timestamps=H5DataIO(self._timestamps, compression='gzip'),
+            description='Pupil size data recorded by EyeLink camera',
+        )
+
+        # Get processing module
+        module_description = 'Contains behavioral data from experiment.'
+        processing_module = get_module(
+            nwbfile=nwbfile, name='behavior', description=module_description)
+
+        # Add data to module
+        processing_module.add_data_interface(pupil_size)
+
+        return nwbfile
+
+
+class RewardLineInterface(TemporalAlignmentMixin):
+    """Reward line interface."""
+
+    def __init__(self, folder_path: FolderPathType):
+        # Find reward line file
+        folder_path = Path(folder_path)
+        reward_line_file = folder_path / 'reward_line.json'
+        assert reward_line_file.exists(), f'Could not find {reward_line_file}'
+        
+        # Load reward line data and set data attributes
+        reward_line_data = json.load(open(reward_line_file, 'r'))
+        self.set_original_timestamps(np.array(reward_line_data['times']))
+        self._reward_line = np.array(reward_line_data['values'])
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+        # Make SpatialSeries
+        reward_line = TimeSeries(
+            name='reward_line',
+            data=H5DataIO(self._reward_line, compression='gzip'),
+            unit='reward line open',
+            timestamps=H5DataIO(self._timestamps, compression='gzip'),
+            description=(
+                'Reward line data representing events of reward dispenser'
+            ),
+        )
+
+        # Get processing module
+        module_description = 'Contains audio and reward data from experiment.'
+        processing_module = get_module(
+            nwbfile=nwbfile, name='misc', description=module_description)
```

Comment:
```suggestion
            nwbfile=nwbfile, name='behavior', description=module_description)
```

Same as the other interface - is there a reason to consider rewards as separate data stream from other behavior?